daisy_workflows/image_import/inspection/boot-inspect.wf.json

```diff
@@ -4,7 +4,8 @@
   "Vars": {
     "pd_uri": {
       "Required": true,
-      "Description": "PD that will be inspected (URI)."
+      "Description": "PD that will be inspected (URI).",
+      "Value": "projects/edens-test/zones/us-west1-b/disks/b194189632"
```

Comment:
Leftover from local testing?

Reply:
shoot, thanks for finding that. Removed!